cmd/list/externalauthprovider/cmd.go

```diff
@@ -79,7 +79,7 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command) error {
 			r.Reporter.Errorf("%s", err)
 			os.Exit(1)
```

Comment:
@chenz4027 Can we also remove this `os.Exit()` and return the `err` instead?

Reply:
Sure, @robpblake I'll work on it